attendance/resources/attendance.py

```diff
@@ -34,8 +45,11 @@ def post(self):
             user = fetchUser(attendance.username)
             if user.get('imageId') is None:
                 compare_faces_facenet(attendance.capturedImageId, attendance.username)
-            else:
+            elif azure_face_enabled == 1:
```

Comment:
Can we have an else block with 2 if conditions for azure and aws? This would allow running both the services in parallel if needed directly from environment variables.